Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt

```diff
@@ -70,12 +70,6 @@ class OnboardingTracingFragment : Fragment(R.layout.fragment_onboarding_tracing)
         }
     }
 
-    override fun onResume() {
-        super.onResume()
-        binding.onboardingTracingContainer.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
```

Comment:
is this container also removed ? why you don't send AccessibilityEvent now?

Reply:
No, I re-added the AccessibilityEvent